diff -rupN JUCE-8.0.12.old/extras/Projucer/Source/Settings/jucer_StoredSettings.cpp JUCE-8.0.12/extras/Projucer/Source/Settings/jucer_StoredSettings.cpp
--- JUCE-8.0.12.old/extras/Projucer/Source/Settings/jucer_StoredSettings.cpp	2025-12-16 09:51:21.000000000 +0100
+++ JUCE-8.0.12/extras/Projucer/Source/Settings/jucer_StoredSettings.cpp	2026-03-30 22:40:36.245814630 +0200
@@ -356,13 +356,13 @@ bool StoredSettings::isJUCEPathIncorrect
 static String getFallbackPathForOS (const Identifier& key, DependencyPathOS os)
 {
     if (key == Ids::jucePath)
-        return (os == TargetOS::windows ? "C:\\JUCE" : "~/JUCE");
+        return (os == TargetOS::windows ? "C:\\JUCE" : "/usr/share/doc/juce");
 
     if (key == Ids::defaultJuceModulePath)
-        return (os == TargetOS::windows ? "C:\\JUCE\\modules" : "~/JUCE/modules");
+        return (os == TargetOS::windows ? "C:\\JUCE\\modules" : "/usr/share/juce/modules");
 
     if (key == Ids::defaultUserModulePath)
-        return (os == TargetOS::windows ? "C:\\modules" : "~/modules");
+        return (os == TargetOS::windows ? "C:\\modules" : "./local/share/juce/modules");
 
     if (key == Ids::vstLegacyPath)
         return {};
